app/views/admin/digitization_queue_items/_form.html.erb

```diff
@@ -1,7 +1,12 @@
 <%= simple_form_for(@admin_digitization_queue_item) do |f| %>
+
+  <% if !@admin_digitization_queue_item.persisted? && @admin_digitization_queue_item.r_and_r_item_id %>
+    <%= hidden_field_tag "admin_digitization_queue_item[r_and_r_item_id]", @admin_digitization_queue_item.r_and_r_item_id  %>
+  <% end %>
```

Comment:
can you put in a comment explaining why this is needed?

Reply:
Done.